site-src/index.md

```diff
@@ -94,8 +94,32 @@ This flexibility allows the API to adapt to vastly different
 organizational models and implementations while remaining a portable and
 standard API.
 
+## What's the difference between Gateway API and an API Gateway?
 
-## Who is working on Gateway?
+The concept of a Gateway is a networking component that allows data to transfer from one
+network to another network.
+
+An API Gateway is a general concept that describes anything that exposes capabilities of a backend service, while providing extra capabilities for traffic routing and manipulation, such as load balancing, request and response transformation, and sometimes more advanced features like authentication and authorization, rate limiting, and circuit breaking.
+
+Most Gateway API implementations are API Gateways to some extent, but not all API Gateways are Gateway API implementations.
+
+Gateway API is an interface, or set of resources, that model service networking
+in Kubernetes. One of the main resources is a `Gateway`, which declares the
+Gateway type (or class) to instantiate and its configuration. As a Gateway Provider,
+you can implement the Gateway API to model Kubernetes service networking in an
+expressive, extensible, and role-oriented way.
+
+Some API Gateways implement Gateway API, while a lot of API Gateways provide
+their own proprietary specification for managing and configuring them.
+
+## Is Gateway API a standard for API Management?
+
+No. API Management is a much broader concept than what Gateway API aims to
+be, or what an API Gateway is intended to provide.
+An API Gateway can be an essential part of an API Management solution.
+Gateway API can be seen as a way to standardize on that aspect of API Management.
```

Comment:
Both of these questions likely belong on our FAQs page. I think the first question "What's the difference between Gateway API and an API Gateway?" is common enough to belong in both places, but this one feels like it should just be on the FAQs page.